microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 

```

Comment:
Global: 
change:
 "ICoreWebView2CompositionController3."
to:
ICoreWebView2CompositionController3::"
(4x)
per C++ stylistic industry convention for Ref docs & per syntax